site/docs/latest/cookbooks/PartitionedTopics.md

```diff
@@ -91,7 +91,7 @@ String topic = "persistent://my-property/my-cluster-my-namespace/my-topic";
 
 PulsarClient client = PulsarClient.create(pulsarBrokerRootUrl);
 ProducerConfiguration config = new ProducerConfiguration();
-config.setMessageRouter(AlwaysTenRouter);
+config.setMessageRouter(new AlwaysTenRouter());
```

Comment:
Don't worry about `site/` docs since these are not used anymore and are on their way of getting removed from repo.

Reply:
Sure, will exclude them in next patch